### PR TITLE
feat: Parametrize gardener shoot converter configuration

### PR DESCRIPTION
**Pull request by m00g3n**
**Description**

Parametrize gardener shoot converter configuration

**Related issue(s)**

- https://github.com/kyma-project/infrastructure-manager/issues/206

### CODE REVIEW

**Review by mvshao — COMMENTED** [review]
Could you add the `feat:` prefix to your PR?

File: cmd/main.go [hunk 1]
```diff
@@ -83,6 +87,7 @@ func main() {
 	flag.DurationVar(&gardenerRequestTimeout, "gardener-request-timeout", defaultGardenerRequestTimeout, "Timeout duration for requests to Gardener")
 	flag.BoolVar(&enableRuntimeReconciler, "runtime-reconciler-enabled", defaultRuntimeReconcilerEnabled, "Feature flag for all runtime reconciler functionalities")
 	flag.BoolVar(&persistShoot, "persist-shoot", false, "Feature flag to allow persisting created shoots")
+	flag.StringVar(&converterConfigFilepath, "converter-config-filepath", "converter_config.json", "A file path to the gardener shoot coverter configuration.")
```
Comment (mvshao) [inline cmd/main.go:90]: The `converter_config.json` is not present in this PR. Should it be `converter_config.yaml` instead?

Comment (m00g3n) [reply cmd/main.go:90]: Thx,
- I will add the configuration to hack dir
- Fix the typo in the description

File: cmd/main.go [hunk 2]
```diff
@@ -83,6 +87,7 @@ func main() {
 	flag.DurationVar(&gardenerRequestTimeout, "gardener-request-timeout", defaultGardenerRequestTimeout, "Timeout duration for requests to Gardener")
 	flag.BoolVar(&enableRuntimeReconciler, "runtime-reconciler-enabled", defaultRuntimeReconcilerEnabled, "Feature flag for all runtime reconciler functionalities")
 	flag.BoolVar(&persistShoot, "persist-shoot", false, "Feature flag to allow persisting created shoots")
+	flag.StringVar(&converterConfigFilepath, "converter-config-filepath", "converter_config.json", "A file path to the gardener shoot converter configuration.")
```
Comment (mvshao) [inline cmd/main.go:90]:
```suggestion
	flag.StringVar(&converterConfigFilepath, "converter-config-filepath", "hack/converter_config.json", "A file path to the gardener shoot converter configuration.")
```

**Review by mvshao — APPROVED** [review]

**Pull request merged**